docs/config.c

```diff
@@ -2739,10 +2740,10 @@
 ** This variable specifies domain name or address of NNTP server.
 ** .pp
 ** You can also specify username and an alternative port for each news server,
-** e.g. \fC[[s]news://][username[:password]@]server[:port]\fP
+** e.g. \fI[[s]news://][username[:password]@]server[:port]\fP
```

Comment:
This is an interesting one.

The [] should be roman, since they are not variable.  They convey meanining by saying that the enclosed text is optional.

Also, some parts are literal, and they should be bold.  This is one of the most complex cases you'll find in a man(7) document, and involves 3 fonts in a single input line.

So, I'd use

```man
.RB [[ s ] news:// ][ \fIusername\fP [ :\fIpassword\fP ] @ ] \fIserver\fP [ :\fIport\fP ]
```

using man(7) macros.  Or the following equivalent with escapes:

```
[[\fBs\fP]\fBnews://\fP][\fIusername\fP[\fB:\fP\fIpassword\fP]\fB@\fP]\fIserver\fP[\fB:\fP\fIport\fP]
```